library/src/interactor_impl.cxx

```diff
@@ -263,6 +263,7 @@ class interactor_impl::internals
       event == vtkF3DConsoleOutputWindow::HideEvent)
     {
       self->Options.ui.console = (event == vtkF3DConsoleOutputWindow::ShowEvent);
+      self->Options.ui.minimal_console = (event == vtkF3DConsoleOutputWindow::ShowEvent);
```

Comment:
This line is not needed at all. The line above is only used when clicking on the console badge when a warning occurs, which in that case is not supposed to open the minimal console but the console.